src/traffic participants/Direction.ts

```diff
@@ -0,0 +1,6 @@
+enum Direction {
```

Comment:
File and folder names must be without spaces. For example, "traffic participants"  better call "trafficParticipants". Let's use the camel case standard